gbpservice/tests/contrib/nfp_scripts/upgrade_nfp.sh

```diff
@@ -0,0 +1,146 @@
+#! /bin/bash
+
+source upgrade_nfp.conf
+source $DEVSTACK_DIR/local.conf
+
+#TODO(DEEPAK): Should be retrieved from a result file populated by advanced mode.
+EXT_NET_NAME=ext-net
+
+function create_port_for_vm {
+    image_name=$1
+
+    GROUP="svc_management_ptg"
+    echo "GroupName: $GROUP"
+    PortId=$(gbp policy-target-create --policy-target-group $GROUP $InstanceName | grep port_id  | awk '{print $4}')
+
+    echo "Getting IpAddr for port: $PortId"
+    IpAddr_extractor=`neutron port-list|grep $PortId|awk '{print $11}'`
```

Comment:
Can we use --format value argument for the neutron commands. This will avoid table symbols in the input.